### PR TITLE
Add remix framework to framework guide

### PR DESCRIPTION
**Pull request by Ekwuno**

### CODE REVIEW

File: products/pages/src/content/framework-guides/remix.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import TutorialsBeforeYouStart from "../_partials/_tutorials-before-you-start.md" 
+
+# Deploy a Remix site
+
+[Remix](https://remix.run/) is a framework that is focused on fully utilizing the power of the web. Like Cloudflare Workers, it uses modern JavaScript APIs, and it places emphasis on web fundamentals such as meaningful HTTP status codes, caching and optimizing for both usability and performance.
+
+In this guide, you will create a new Remix application and deploy to Cloudflare Pages. 
+
+## Setting up a new project 
+Start by installing the latest version of Remix. Create a new project directory and then intialize a Remix project by running:
```
Comment (deadlypants1973) [inline products/pages/src/content/framework-guides/remix.md]:
```suggestion
## Setting up a new project

Start by installing the latest version of Remix. Create a new project directory and then intialize a Remix project by running:
```

File: products/pages/src/content/framework-guides/remix.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+```sh
+$ npx create-remix@latest
+
```
Comment (deadlypants1973) [inline products/pages/src/content/framework-guides/remix.md:19]:
```suggestion
$ npx create-remix@latest
```

File: products/pages/src/content/framework-guides/remix.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+After running the above command, you will be prompted to name your project and select your deploy method. This guide uses Cloudflare Pages. 
+
+By selecting Cloudfalre Pages as your deploy option your folder will have a `functions/[[path]].ts` file. This is the functions integration where we serve your Remix application on all paths of your website. After this change directory to your project and see your project by runnig the following command:
```
Comment (deadlypants1973) [inline products/pages/src/content/framework-guides/remix.md]:
```suggestion
By selecting Cloudflare Pages as your deploy option in the CLI drop down, your folder will have a `functions/[[path]].ts` file. This is the functions integration where we serve your Remix application on all function paths of your website. After selecting your deployment option, change the directory to your project and see your project by running the following command: 
```

1) Can this part be more clear? How does the user select CF Pages? Is there a menu? Or do they type it in?
2) What is the 'functions integration'? The file? Is this saying that CF Pages will use the `functions/[[path]].ts` file to serve the Remix application.
3) Last sentence needs some revision. "After this" -> define "this". Seems to be some word missing the makes the sentence incomplete.

**Review by deadlypants1973 — CHANGES_REQUESTED** [review]

File: products/pages/src/content/framework-guides/remix.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+## Setting up a new project
+
+Start by installing the latest version of Remix. Create a new project directory and then intialize a Remix project by running:
+
+```sh
+$ npx create-remix@latest
+
+```
+
+After running the above command, you will be prompted in the CLI to name your project and select your deploy method. This guide uses Cloudflare Pages.
```
Comment (deadlypants1973) [inline products/pages/src/content/framework-guides/remix.md]:
```suggestion
After running the above command, you will be prompted to name your project in your terminal and select your deploy method. This guide uses Cloudflare Pages. 
```

File: products/pages/src/content/framework-guides/remix.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+Start by installing the latest version of Remix. Create a new project directory and then intialize a Remix project by running:
+
+```sh
+$ npx create-remix@latest
+
+```
+
+After running the above command, you will be prompted in the CLI to name your project and select your deploy method. This guide uses Cloudflare Pages. 
+
+By selecting Cloudflare Pages as your deploy option in the CLI drop down, your folder will have a `functions/[[path]].ts` file. This is the functions integration where we serve your Remix application on all function paths of your website. After selecting your deployment option, change the directory to your project and see your project by running the following command:
```
Comment (deadlypants1973) [inline products/pages/src/content/framework-guides/remix.md]:
```suggestion
By selecting *Cloudflare Pages* as your deployment option in the terminal drop-down menu, your folder will have a `functions/[[path]].ts` file. This file is the functions integration where Cloudflare serves your Remix application on all function paths of your website. After selecting your deployment option, change the directory to your project and render your project by running the following command: 
```

File: products/pages/src/content/framework-guides/remix.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+Start by installing the latest version of Remix. Create a new project directory and then intialize a Remix project by running:
+
+```sh
+$ npx create-remix@latest
+
+```
+
+After running the above command, you will be prompted to name your project in your terminal and select your deploy method. This guide uses Cloudflare Pages. 
+
+By selecting *Cloudflare Pages* as your deployment option in the terminal drop-down menu, your folder will have a `functions/[[path]].ts` file. This file is the functions integration where Cloudflare serves your Remix application on all function paths of your website. After selecting your deployment option, change the directory to your project and render your project by running the following command:
```
Comment (lukeed) [inline products/pages/src/content/framework-guides/remix.md]:
```suggestion
By selecting *Cloudflare Pages* as your deployment option in the terminal drop-down menu, your folder will have a `functions/[[path]].ts` file. The `[[path]]` filename indicates that this file will handle requests to all incoming URLs; refer to [Path segments](https://developers.cloudflare.com/pages/platform/functions#path-segments) to learn more.  

After selecting your deployment option, change the directory to your project and render your project by running the following command: 
```

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**